index.html

```diff
@@ -1,45 +1,451 @@
+                    Once selected, the source of each {{MediaStreamTrack}} MUST NOT change,
+                    unless the user permits it through their interaction with
+                    the <a href="#dfn-user-agent">user agent</a>.</p>
+                  <p><a href="#dfn-user-agent">user agents</a> are encouraged to warn
+                    users against sharing browser <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>
+                    as well as monitor <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>
+                    where browser windows are visible, or
+                    otherwise try to discourage their selection on the basis
+                    that these represent a significantly higher risk when shared.</p>
+                  <p>If the result of the request is {{PermissionState/"granted"}}, then for
+                    each device that is sourcing the provided medias, using
+                    a stable and private id for the device, <var>deviceId</var>,
+                    set [[\devicesLiveMap]]<var>[deviceId]</var> to
+                    <code>true</code>, if it isn’t already <code>true</code>,
+                    and set the
+                    [[\devicesAccessibleMap]]<var>[deviceId]</var> to
+                    <code>true</code>, if it isn’t already
+                    <code>true</code>.
+                  </p>
+                  <p>
+                    The user agent MUST NOT store a {{PermissionState/"granted"}} permission entry.
+                  </p>
+                  <p>If the result is {{PermissionState/"denied"}}, jump to the step labeled
+                    <em>Permission Failure</em> below. If the user never
+                    responds, this algorithm stalls on this step.
+                  </p>
+                  <p>If the user grants permission but a hardware error
+                    such as an OS/program/webpage lock prevents access to at least one device,
+                    <a>reject</a> <var>p</var> with a new
+                    {{DOMException}} object whose
+                    {{DOMException/name}} attribute has the value
+                    {{NotReadableError}} and abort these steps.
+                  </p>
+                  <p>If the result is {{PermissionState/"granted"}} but device access fails for
+                    any reason other than those listed above, <a>reject</a>
+                    <var>p</var> with a new {{DOMException}}
+                    object whose {{DOMException/name}} attribute has the
+                    value {{AbortError}} and abort these steps.
+                  </p>
+                </li>
+                <li>
+                  <p>Let <var>streams</var> be the
+                    list of {{MediaStream}} objects for which the user granted permission.</p>
+                </li>
+                <li>
+                  <!-- Guidance taken from https://infra.spec.whatwg.org/ -->
```

Comment:
That's a long document. Which specific part of it?

Reply:
It's gone.